src/index.html

```diff
@@ -1,15 +1,110 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST"
+    autocomplete="off"
+    onsubmit="onSubmit()">
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <div>
+          <label for="surname">Surname:</label>
+          <input type="text" id="surname">
+          <br>
+        </div>
+        <div>
+          <label for="name">Name:</label>
+          <input type="text" id="name">
+        </div>
+        <div>
+          <label for="age">How old are you?</label>
+          <input type="number" id="age"
+          value="12" min="1"
+          max="100">
```

Comment:
```suggestion
          <input 
            type="number" 
            id="age"
            value="12" min="1"
            max="100"
          >
```
отредактируй так везде